mlshvch/hometask-1/hello_world.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
```

Comment:
This won't make the file executable.
Try to run `$ ./hello_world.rb` in the hometask folder.

Reply:
Or, to be precise, that's not _the only_ thing to make it executable.

Reply:
Do you mean adding information about "chmod +x /mlshvch/hometask-1/hello_world.rb" into README.md or sth else?

And as I've understood from previous comments, I need to revert state of branch to "Create hello_world.tb file" commit by means of "git revert"?

Reply:
@StalemateInc @mlshvch yes, but this part especially was done with typo which makes me thinking it wasn't tested locally

Reply:
@mlshvch
`This won't make the file executable.` means your file is not executable.
```
➜  hometask-1 git:(mlshvch-hometask-1) ✗ ./hello_world.rb
./hello_world.rb: line 5: puts: command not found
```
You have a typo in line 3.

Speaking of commit history, you've got a bunch of options.
Some are easy to do, some are pretty complex, but I'm going to suggest the simplest one.

In my opinion for this particular PR base on changes you've made, your commit history should look like:
- Create hello_world.rb file
- Make hello_world.rb file executable
- Added frozen_string comment to hello_world.rb file
- Fixed comments in hello_world.rb file

or you a bit simpler:
- Create hello_world.rb file
- Make hello_world.rb file executable
- Fix Hound violations

But as soon as you haven't completed `executable` part of the task, in your case, I'd suggest to:
- complete the task
- run rubocop locally in order to avoind Hound violations
- squash commits in one locally and it'll 1 commit history